_docs/timeline/2017-01-02-bootstrap.md

```diff
@@ -22,3 +22,40 @@ The Bootstrap era will lead to the [Reward era](/timeline/reward), during which
 updates to the protocol will be issued, and the major stakeholders will be
 provided with convenient options to run their nodes on personal servers in the
 cloud.
+
+## Stake Locking
+
+Bootstrap era is the period of Cardano SL existence that allows only fixed predefined
+users to have control over the system. The set of such users (bootstrap stakeholders)
+is defined in [`gcdBootstrapStakeholders`](https://github.com/input-output-hk/cardano-sl/blob/f73d41b2bbd0a823911490974a71608fb4dbe014/core/Pos/Core/Genesis/Types.hs#L74). Bootstrap era will end when bootstrap stakeholders
+will vote for it. Next era after Bootstrap is called Reward era, as mentined [here](/timeline/reward/).
```

Comment:
Need to add details on how voting will happen:

Special update proposal will be formed, where particular constant will be set appropriately (how, BTW, need a reference to this constant in source file) to trigger boot era end at the point update proposal gets adopted.